tests/unittests/CMakeLists.txt

```
@@ -0,0 +1,135 @@
+# Enable Qt MOC/UIC/RCC
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTOUIC OFF)
+set(CMAKE_AUTORCC OFF)
+
+# Path to test data
+add_definitions(-DTEST_DATA_DIR="${CMAKE_CURRENT_SOURCE_DIR}/../data")
+
+# Tests require libpthread
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
+
+# Main executable
+add_executable(librepcb_unittests
+    # Sources
+    common/algorithm/airwiresbuildertest.cpp
+    common/alignmenttest.cpp
+    common/applicationtest.cpp
+    common/attributes/attributekeytest.cpp
+    common/attributes/attributesubstitutortest.cpp
+    common/attributes/attributetest.cpp
+    common/attributes/attributetypetest.cpp
+    common/attributes/attributeunittest.cpp
+    common/boarddesignrulestest.cpp
+    common/cam/gerberaperturelisttest.cpp
+    common/cam/gerberattributetest.cpp
+    common/cam/gerberattributewritertest.cpp
+    common/cam/gerbergeneratortest.cpp
+    common/circuitidentifiertest.cpp
```

This was lost during rebase:

```suggestion
    common/circuitidentifiertest.cpp
    common/fileio/asynccopyoperationtest.cpp
```

---

Good catch!